packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -184,6 +185,10 @@ class InspectorTreeController extends Object
   final List<InspectorTreeRow> cachedRows = [];
   InspectorTreeRow _cachedSelectedRow;
 
+  /// Cached rows of the tree. Similar to [cachedRows] but items are populated
```

Comment:
dart-doc nit: there should be a blank new line between the first sentence and the rest. 

Also just to clarify, `_searchableCachedRows` contains every row in the tree, is that correct? not just those that are visible (nodes with expanded parents)?

Reply:
Yes, it contains every row in the tree (including collapsed nodes, children of collapsed nodes), it's now reflected in the doc